core/src/main/java/sem/group15/bubblebobble/core/objects/SimpleEnemy.java

```diff
@@ -19,32 +19,10 @@ public SimpleEnemy(final float xPosition, final float yPosition) {
     }
 
 
-
-    /**
-     * Handles collisions with other objects.
-     * If there is a WallCollision set the attribute wallCollision to true.
-     * @param other Object that needs to be checked for collision.
-    */
     @Override
     public void handleCollision(GameObject other) {
         super.handleCollision(other);
```

Comment:
If you just call super, no need to overwrite and just let superclass handle it, right?